### DISCUSSION TITLE
Error in creating the link along with domain or customurl

### DISCUSSION THREAD

**Discussion by Biswadeepgiri** [thedevs-network/kutt#758] [General]
When I am hitting the api from an external endpoint to create the link , when I am only passing the target then the url is being shortened but when I am passing domain or customurl along with the target then I am getting an error

 data: { error: 'Only users can use this field.' }
 
 please tell how to solve this issue. @poeti8

**Comment by poeti8** [comment]
You should pass an API key. Grab the API key from the settings page for the user.
Without an API key the link creation is anonymous. Custom URL is not available for anonymous users.
